about.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Jenny Luong Portfolio</title>
+  <link rel="stylesheet" href="stylesheets/style-index.css">
+  <link rel="stylesheet" href="stylesheets/style-about.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+  <section id="container-about">
+    <header>
+      <h1>Jenny Luong</h1>
+      <h3> &lt;Software Developer /&gt; </h3>  
+      <h3>Occupational Therapist</h3>
+      <nav>
+        <a href="index.html">Home</a>
+        <a href="about.html">About Me</a>
+        <a href="portfolio.html">Portfolio</a>
+      </nav>
+    </header>
+
+    <section id="intro">
+      <img src="../assets/jen-at-laptop.jpg" alt="Jenny working on laptop">
+      <p> Hi, I'm Jenny (she/her). <br> <br> I am delving into a new area of learning: programming languages, computer science principals, logical thinking, and creating with technology.<br><br>After years of working as an occupational therapist, teaching others about neuroplasticity and our abilities to rehabilitate and remake ourselves, I felt empowered to develop a new set of skills within myself.</p>
```

Comment:
Style: Instead of using `br` tags here you should make these separate paragraphs:

```suggestion
      <p> Hi, I'm Jenny (she/her).</p>
      <p>I am delving into a new area of learning: programming languages, computer science principles, logical thinking, and creating with technology.</p>
      <p>After years of working as an occupational therapist, teaching others about neuroplasticity and our abilities to rehabilitate and remake ourselves, I felt empowered to develop a new set of skills within myself.</p>
```